content/blog/getting-started-with-pac/index.md

```diff
@@ -0,0 +1,172 @@
+
+<!--more-->
+
+## Leaky  Buckets
+
+What do many of the most innovative companies in the modern cloud era have in common? They’ve all had massive data breaches from leaky AWS S3 Buckets.  One of the most common reasons for data breaches is misconfigured AWS S3 Buckets. Both security professionals and bad actors use tools such as slurp to find open S3 buckets by just entering a domain, keyword, or even AWS credentials with the slurp CLI. Ideally, we want to proactively prevent public access to our S3 buckets before they are created.
+
+![S3 buckets open to public](Slurp-Amazon-AWS-S3-Bucket-Enumerator.png)
+
+Pulumi's CrossGuard helps customers adhere to security best practices by checking the final state of a deployment against a set of policies and revealing any non-compliant services. CrossGuard is an SDK for writing policies for services, and it includes best practices playbooks for cloud providers such as AWS, Azure, and Google Cloud Platform. The [Getting Started with PaC]({{< ref "/docs/get-started/crossguard" >}}) covers the case of leaking S3 buckets, but let’s do a quick review with an example showing how we can detect a leaky bucket before it gets deployed.
+
+>Note: If you haven’t installed Pulumi, follow the [Pulumi Getting Started]({{< ref "/docs/get-started/install" >}})
+Start by creating an S3 bucket with Pulumi.
+
+```bash
+$ pulumi new aws-typescript
+```
+
+ Running `pulumi new` creates the project files as well as the code to create a single S3 bucket. We’ll use an S3 ACL to make the bucket public by setting it to the [canned ACL `public-read`](https://docs.aws.amazon.com/AmazonS3/latest/dev/acl-overview.html#canned-acl).
+
+```ts
+const bucket = new aws.s3.Bucket("my-bucket", {
+    acl: "public-read",
+});
+```
+
+Next, we’ll create a Policy Pack, by creating a directory and making it the current working directory.
+
+```bash
+$ mkdir policypack && cd policypack
+```
+
+CrossGuard is still in *preview*, and we need to set the environmental variable `PULUMI_EXPERIMENTAL` to use it. We can prepend it before creating the policy.
+
+```bash
+$ PULUMI_EXPERIMENTAL=true pulumi policy new aws-typescript
+```
+
+By default, this creates a policy that checks to see if the ACL sets the bucket to be publically accessible.
+
+```ts
+new PolicyPack("aws-typescript", {
+    policies: [{
+        name: "s3-no-public-read",
+        description: "Prohibits setting the publicRead or publicReadWrite permission on AWS S3 buckets.",
+        enforcementLevel: "mandatory",
+        validateResource: validateTypedResource(aws.s3.Bucket, (bucket, args, reportViolation) => {
```

Comment:
Note: We recently renamed `validateTypedResource` to `validateResourceOfType` (though, `validateTypedResource` can still be used -- it's just marked as deprecated).

If this is published after https://github.com/pulumi/templates-policy/pull/15 is merged, we should change it here:

```suggestion
        validateResource: validateResourceOfType(aws.s3.Bucket, (bucket, args, reportViolation) => {
```

Otherwise, I'll come back and take care of updating the blog post after https://github.com/pulumi/templates-policy/pull/15 is merged.